### ISSUE TITLE
ref is failing with 8.3.0

### ISSUE THREAD

**Issue by al-k21** [nodejs/citgm#464]
```bash
error:                 | 132 passing (176ms)                                                                                                                                         
error:                 | 2 failing                                                                                                                                                   
error:                 |                                                                                                                                                             
error:                 | 1) Object should retain references to written Objects:                                                                                                      
error:                 |                                                                                                                                                             
error:                 | AssertionError [ERR_ASSERTION]: "o" has not been garbage collected                                                                                          
error:                 | + expected - actual                                                                                                                                         
error:                 |                                                                                                                                                             
error:                 | -false                                                                                                                                                      
error:                 | +true                                                                                                                                                       
error:                 |                                                                                                                                                             
error:                 | at Context.<anonymous> (test/object.js:46:5)                                                                                                                
error:                 |                                                                                                                                                             
error:                 | 2) pointer should retain references to a written pointer in a Buffer:                                                                                       
error:                 |                                                                                                                                                             
error:                 | AssertionError [ERR_ASSERTION]: "child" has not been garbage collected                                                                                      
error:                 | + expected - actual                                                                                                                                         
error:                 |                                                                                                                                                             
error:                 | -false                                                                                                                                                      
error:                 | +true                                                                                                                                                       
error:                 |                                                                                                                                                             
error:                 | at Context.<anonymous> (test/pointer.js:45:5)                                                                                                               
```

* **Node Version**: 8.3.0 (does not fail on 8.2.1)
* **Citgm version**: 2.0.5
* **Platform**: Darwin

**Comment by al-k21** [comment]
@TooTallNate do you know why is it failing ?

**Comment by targos** [comment]
This is likely due to the V8 upgrade. We have to investigate if this is a bug in the garbage collector or a wrong assumption from the test.

**Comment by targos** [comment]
Attempt to fix the tests: https://github.com/TooTallNate/ref/pull/85

**Comment by targos** [comment]
This should be fixed with `ref` v1.3.5.

**Comment by gdams** [comment]
confirmed fixed for me
